### ISSUE TITLE
NVDA is again frequently unable to gain focus when alt-tabbing back into VS Code

### ISSUE THREAD

**Issue by Neurrone** [microsoft/vscode#87154]
Issue Type: <b>Bug</b>

CC: @isidorn @bpasero

This is unfortunately a resubmission of #28316.

This only started happening again for me in the last few insiders build, so I'm wondering what changed? Its also possible that an update that I did to recent NVDA betas caused this, though I feel this is less likely.

NVDA is (quite frequently) unable to focus on the control with keyboard focus in VS code when alt tabbing into it from another application. 

When this occurs, Code is completely unuseable with NVDA (pressing tab and arrow keys doesn't do anything)

VS Code version: Code - Insiders 1.42.0-insider (2d2ac8f75b539d505443ac0733b196851c0b218d, 2019-12-16T12:09:59.453Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz (4 x 2904)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.89GB (3.68GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (9)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.0.5
gitlens|eam|10.2.0
prettier-vscode|esb|3.14.0
rest-client|hum|0.23.0
Go|ms-|0.11.9
vscode-typescript-tslint-plugin|ms-|1.2.3
vsliveshare|ms-|1.0.1377
vscode-yaml|red|0.6.1
rust|rus|0.7.0

</details>

**Comment by bpasero** [comment]
@Neurrone please double check that you can not reproduce in stable but reproduce in insiders. And if so, we can try to find the version where this started maybe?

Giving a few insider builds to play with:
* today: https://az764295.vo.msecnd.net/insider/22a87bf3f259d3fcff66f636a9db611741c7e1c5/VSCode-win32-x64-1.42.0-insider.zip
* 1d ago: https://az764295.vo.msecnd.net/insider/2d2ac8f75b539d505443ac0733b196851c0b218d/VSCode-win32-x64-1.42.0-insider.zip
* last week: https://az764295.vo.msecnd.net/insider/cbeff45f80213db0ddda2183170281ed97ed3b12/VSCode-win32-x64-1.42.0-insider.zip

**Comment by isidorn** [comment]
@Neurrone can you also try reproducing with the exploration build which uses Electron 7 https://az764295.vo.msecnd.net/exploration/22c3db00ec057fecf0144c4c7f8c6bc6cb60e5d6/VSCodeSetup-x64-1.41.0-exploration.exe

Adding important label since this is a pain and that we make sure it does not fall off the radar.

**Comment by bpasero** [comment]
Re exploration builds: they are quite old, I am currently running newer builds with latest (they are coming with Electron 7.1.5).

**Comment by Neurrone** [comment]
The Last week build was fine: 

```
Commit: cbeff45f80213db0ddda2183170281ed97ed3b12
Date: 2019-12-13T07:08:57.560Z
```

I experienced this issue with the build you labelled "yesterday" and "today". But the even newer snapshot below fixed it for me:

```
Commit: 241d4048aabc31db0baed66bb3d58cf3210d981d
Date: 2019-12-17T10:43:45.626Z
```

So something between "last week" and "yesterday" caused the issue, which got fixed somewhere between "today" and the most recent one.

**Comment by Neurrone** [comment]
If that doesn't provide enough clues to narrow down the cause, I can also test out each daily build between "last week" and "yesterday" if you point me to where I can find a sorted list of previous snapshot archives.

**Comment by isidorn** [comment]
@Neurrone thank you very much. So it seems like the latest insiders works just fine.
So I belive the issue you have seen was caused by the change to `backgroundThrottling` flag https://github.com/microsoft/vscode/commit/e2dbfd4b974487474f65ab54189531f97d8281e2

Since this works in latest I think we can close this. Though leaving it up to @bpasero

**Comment by Neurrone** [comment]
Is there anywhere that I can find more info on this flag? Is it an electron feature? Am wondering if this issue could also happen in other Electron apps, or just Code.

**Comment by Neurrone** [comment]
Also, this issue existed for a long time until fairly recently (some months back). Was this flag present as well then? Or was this something added only recently?

**Comment by bpasero** [comment]
@Neurrone it is a patch Electron maintains and we noticed issues with it (windows blanking out). The patch is: https://github.com/electron/libchromiumcontent/blob/master/patches/common/chromium/disable_hidden.patch

**Comment by bpasero** [comment]
And see the property mentioned here: https://github.com/electron/electron/blob/6-0-x/docs/api/browser-window.md
